compile_system/statement.py

```diff
@@ -1,24 +1,30 @@
+from writeFile import getInputPath, getOutputPath
+
 def constructCompileStatement(target, language, fileName):
     
     command = determineCompiler(target, language) + determineTarget(target)
     staticLinkParam = determineStaticLibraryLink(target, language)
 
-    outputFileName = fileName
-    outputFileName = outputFileName.strip("." + language)
+    inputPath = getInputPath(fileName)
+
+    outputFileName = fileName.strip("." + language)
+    outputPath = getOutputPath(outputFileName)
 
-    removeCommand = "rm -rf " + fileName
+    removeCommand = "rm -rf " + inputPath + " " + outputPath 

```

Comment:
Should only remove input file, removing the output file deletes the executable immediately after compilation

Reply:
Dont worry about deleting the compiled file